ox-textile.el

```diff
@@ -146,9 +146,13 @@ CONTENTS is the contents of the list.  INFO is a plist holding
 contextual information."
   contents)
 
-(defun org-textile-item-list-depth (item)
-  (let ((parent item)
-	(depth 0))
+(defun org-textile-item-list-depth (item info)
+  (let* ((headline-level
+	  (org-export-get-relative-level
+	   (org-export-get-parent-headline item) info))
+	 (headline-levels (plist-get info :headline-levels))
+	 (parent item)
+	 (depth (max 0 (- headline-level headline-levels))))
```

Comment:
You can get the depth using `org-export-low-level-p`.  How about something like this:

```elisp
(let* ((parent item)
         (headline (org-export-get-parent-headline item))
	 (depth (or (and headline
                         (org-export-low-level-p headline info))
                    0)))
```
Is this feasible?